modules/cloudflarepurger/src/Plugin/Purge/Purger/CloudFlarePurger.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\cloudflare\Plugin\PurgePurger\CloudFlarePurger.
+ */
+
+namespace Drupal\cloudflarepurger\Plugin\Purge\Purger;
+use Drupal\Core\Cache\Cache;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\cloudflare\CloudFlareStateInterface;
+use Drupal\cloudflarepurger\EventSubscriber\CloudFlareCacheTagHeaderGenerator;
+use Drupal\purge\Plugin\Purge\Purger\PurgerBase;
+use Drupal\purge\Plugin\Purge\Purger\PurgerInterface;
+use Drupal\purge\Plugin\Purge\Invalidation\InvalidationInterface;
+use CloudFlarePhpSdk\ApiEndpoints\CloudFlareAPI;
+use CloudFlarePhpSdk\ApiEndpoints\ZoneApi;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * CloudFlare purger.
+ *
+ * @PurgePurger(
+ *   id = "cloudflare",
+ *   label = @Translation("CloudFlare"),
+ *   description = @Translation("Purger for CloudFlare."),
```

Comment:
Perhaps a slightly richer description? Again, nitpicking, this is by far MVP already.